### PR TITLE
Download improvements 

### PR DESCRIPTION
**Pull request by patrickelectric**
This PR tries to minimize issues related to SSL.
- Move urls from https to http.
- Ignore SSL errors while downloading data with QGCFileDownload.
- Print human-readable description of SSL errors.

### CODE REVIEW

**Comment by DonLakeFlyer** [comment]
Can you please provide way more detail with your pull requests? What exactly are you trying to fix here would be helpful? Are you trying to work around this: https://github.com/mavlink/qgroundcontrol/pull/6354 which has already been fixed?

File: src/QGCFileDownload.cc [hunk 1]
```diff
@@ -76,6 +76,7 @@ bool QGCFileDownload::download(const QString& remoteFile, bool redirect)
     networkRequest.setAttribute(QNetworkRequest::User, localFile);
 
     QNetworkReply* networkReply = get(networkRequest);
+    networkReply->ignoreSslErrors();
```
Comment (DonLakeFlyer) [inline src/QGCFileDownload.cc]: I think @dogmaphobic deals with this sort of things in map tiles differently. Would be good to be consistent with that. @dogmaphobic ?

**Comment by patrickelectric** [comment]
Hi @DonLakeFlyer, Sorry for not being so clear. Description updated.

**Comment by patrickelectric** [comment]
Probably:
- QGCFileDownload: Continue download if a SSL error happens  … f6a6c6d
- QGCFileDownload: Print SSL error and SSL Certificate error  … 3fd4d14
Are the best one to continue with this patch, the others just try to minimize any future problem, if there is any.

**Comment by DonLakeFlyer** [comment]
I think @dogmaphobic Has done something different what that to just ignore cert errors for map tiles for examples. Gus?

**Comment by DonLakeFlyer** [comment]
Most of the other SSL stuff is done now. I'm going to pull out the changes to File Download ssl error stuff and create a separate pull for that.

**Comment by patrickelectric** [comment]
Ty Don, let me know if I can help to test it.

**Comment by jaxxzer** [comment]
Is any part of this still relevant? Should we rebase to get the human-readable error messages?

**Comment by DonLakeFlyer** [comment]
I don't think so. But that said SSL doesn't work on latest Ubuntu due to the Qt/SSL strangeness.

**Pull request closed without merging**